File: TestHello/TestHello/ViewController.m
//
//  ViewController.m
//  TestHello
//
//  Created by Fei on 16/1/20.
//  Copyright © 2016年 WangQiaoFei. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
- (IBAction)btnAdd:(id)sender;

- (IBAction)btnSave:(id)sender;
@property (weak, nonatomic) IBOutlet UILabel *lbShowMsg;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)btnAdd:(id)sender {
    self.lbShowMsg.text = @"点击add";
}

- (IBAction)btnSave:(id)sender {
    self.lbShowMsg.text = @"点击save";
}
@end

File: TestFirst/TestFirst/ViewController.swift
//
//  ViewController.swift
//  TestFirst
//
//  Created by Fei on 15/12/9.
//  Copyright © 2015年 WangQiaoFei. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    var clickTimes = 0 ;
    @IBOutlet weak var lable1: UILabel!
    
    @IBAction func onClick(sender: AnyObject) {
        clickTimes++
        self.lable1.text = "clicked \(clickTimes) times"
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

